templates/projects.ini.erb

```diff
@@ -9,4 +9,6 @@ target=<%= @target %>
 smtpserver=<%= @dlrn_mailserver %>
 reponame=delorean
 tags=<%= @release %>
+rsyncdest=<%= @rsyncdest %>
```

Comment:
Let's use the same conditional statement we're using for gerrit:
<% if @rsyncdest %>...<% end %>